cmake/modules/CompileShaders.cmake

```diff
@@ -3,7 +3,10 @@ function(compile_shaders)
     set(OUTPUT_FILE_NAME "${CMAKE_CURRENT_BINARY_DIR}/${X}.h")
     set(INPUT_FILE_NAME "${CMAKE_CURRENT_SOURCE_DIR}/${X}")
 
+    get_filename_component(OUTPUT_DIR ${OUTPUT_FILE_NAME} DIRECTORY)
+
     add_custom_command(OUTPUT "${OUTPUT_FILE_NAME}"
+      COMMAND ${CMAKE_COMMAND} -E make_directory ${OUTPUT_DIR}
```

Comment:
Seems like a perfectly reasonable change, although I am puzzled by the fact that I did not need it on Debian nor did Barbara need it on Windows. Do you have any idea why this broke for you?

Reply:
No clue, but the build broke without it. It resulted in a compiler error saying the headers could not be found. With this change, all is well :D.